php/helpers.php

```diff
@@ -165,7 +165,17 @@ function block_field_config( $name ) {
  * @return string|array
  */
 function block_lab_locate_template( $template_names, $path = '', $single = true ) {
-	$path            = apply_filters( 'block_lab_template_path', $path );
+	/**
+	 * Filters the path where block templates are saved.
+	 *
+	 * Note that template names are prefixed with the blocks directory.
+	 * e.g. `blocks/block-template.php`
+	 *
```

Comment:
Maybe this could also mention:

```
 * The logic below will look for the prefixed template name inside the $path.
```

Reply:
Great idea. Added in https://github.com/getblocklab/block-lab/pull/358/commits/4cc09669bd1bb6343cbeed438b37c4fc96b1a52c.